Source/WebGPU/WebGPU/USDModel.swift

```diff
@@ -1049,146 +1154,209 @@ extension DDBridgeReceiver {
     }
 }
 
-final class Converter {
```

Comment:
'ignore whitespace' setting in Githubs helps reduce the change markers, since I just moved these to freestanding functions